api-spec/protobuf/ark/v1/service.proto

```diff
@@ -89,6 +95,15 @@ message CompletePaymentRequest {
 }
 message CompletePaymentResponse {}
 
+message ReverseBoardingAddressRequest {
+  string pubkey = 1;
+}
+
+message ReverseBoardingAddressResponse {
+  string address = 1;
+
+}
+
```

Comment:
Its' try is handy to have server to generate the address, but we also need to give possibility to build the script "offline" assuming user after first connection caches the descriptor template accepted (ideally we could have an expiry time also attached to each descriptor template)